src/gui/MainWindow.h

```diff
@@ -310,6 +310,7 @@ private slots:
 
   QList<double> getTranslation() const;
   QList<double> getRotation() const;
+  std::unordered_map<QAction*, FileFormat>  export_map;
```

Comment:
I'm wondering if this map should be reversed, to allow us to look up an action from a file format.